### PR TITLE
add small modal header option

### PR DESCRIPTION
**Pull request by garrettbear**

### CODE REVIEW

File: components/atoms/ModalHeader.jsx [hunk 1]
```diff
@@ -17,7 +19,8 @@ ModalHeader.propTypes = {
     title: PropTypes.string,
     onClose: PropTypes.func,
     className: PropTypes.string,
-    style: PropTypes.object
+    style: PropTypes.object,
+    size: PropTypes.string
```
Comment (jscottsmith) [inline components/atoms/ModalHeader.jsx]:
```
size: PropTypes.oneOf( 'sm']),
```
so it's clear that's the only accepted size

File: components/atoms/ModalHeader.jsx [hunk 2]
```diff
@@ -2,9 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-const ModalHeader = ({ title, onClose, className, style }) => (
-    <div className={cx('gds-modal__header', className)} style={style}>
-        {title && <h2 className="gds-modal__title gds-text--header-sm">{title}</h2>}
+const ModalHeader = ({ title, onClose, className, style, size }) => (
+    <div className={cx('gds-modal__header',
+    {'gds-modal__header--sm': size === 'sm'},
```
Comment (jscottsmith) [inline components/atoms/ModalHeader.jsx]: doesn't look like prettier was run on this...

File: components/atoms/ModalHeader.jsx (hunk 1 above)
Comment (garrettbear) [reply components/atoms/ModalHeader.jsx]: ahhh cooo.

**Comment by garrettbear** [comment]
Stable is up to date and working well,  @mnicole clear your cache on your browser. That's most likely the issue.

**Comment by mnicole** [comment]
> Stable is up to date and working well, @mnicole clear your cache on your browser. That's most likely the issue.

good point, thanks! 😄

**Pull request merged**